components/popover/popover.vue

```diff
@@ -823,6 +824,11 @@ export default {
         this.$refs.content.$el.focus();
       }
     },
+
+    handleWheel (e) {
+      e.preventDefault();
+      console.log(e);
+    },
```

Comment:
Do you need this?

Reply:
No, forgot to remove it, was using it for testing.